python/ray/data/_internal/arrow_block.py

```diff
@@ -412,9 +412,10 @@ def sum_of_squared_diffs_from_mean(
         )
 
     def sort_and_partition(
-        self, boundaries: List[T], key: "SortKeyT", descending: bool
+        self, boundaries: List[T], sort_key: "SortKey"
     ) -> List["Block"]:
-        if len(key) > 1:
+        columns, ascending = sort_key.to_pandas_sort_args()
```

Comment:
nit, this function is not related to pandas, so it's a bit confusing to call `to_pandas_sort_args`. I think we can call 2 separate APIs to get them.

Reply:
Thanks, added a separate `get_descending()` function, in addition to the existing `get_columns()`.